### PR TITLE
Replace deprecated Litle URLs

### PR DESCRIPTION
**Pull request by grantbdev**
This PR [replaces deprecated Litle URLs with the new Vantiv URLs](https://www.vantiv.com/content/dam/vantiv/merchants-partners/docs/ecommerce-domains-faq_v1-apr-2017.pdf).

### CODE REVIEW

**Comment by grantbdev** [comment]
@tconst Updated this PR to follow the newly rebased `master` and `add-vantiv-gateway` and also update the remote certification tests.

File: lib/active_merchant/billing/gateways/vantiv.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ module Billing #:nodoc:
     # name.
     class VantivGateway < Gateway
       self.test_url = "https://www.testlitle.com/sandbox/communicator/online"
```
Comment (tconst) [inline lib/active_merchant/billing/gateways/vantiv.rb]: Is there an updated test url?

Comment (grantbdev) [reply lib/active_merchant/billing/gateways/vantiv.rb]: There was not at that time, but there is [now](https://developer.vantiv.com/message/3721). Updating.

**Comment by grantbdev** [comment]
@tconst Rebased `add-vantiv-gateway` on latest upstream master and then updated this PR with the bare minimum to keep the tests passing.

**Review by tconst — APPROVED** [review]

**Pull request merged**